documentation/api/query/v4/operators.markdown

```diff
@@ -176,6 +178,30 @@ To get the average uptime for your nodes,
 
     ["extract", [["function", "avg", "value"]], ["=", "name", "uptime_seconds"]]
 
+## Context Operators
+
+*Note:* Context setting support is new and experimental. Setting the context at
+the top of the query is only supported on the [root] endpoint.
+
+Setting context in a query allows you to choose the entity you are querying
+on. This augments the endpoint support we have today, whereby the endpoint
+decides the context by allowing a single endpoint to contain all the information
+required to show results.
+
+### `from`
+
+The `from` operator allows one to choose the entity that you want to query and
```

Comment:
Could `entity` here have a link to the "entities" doc? Channeling my inner Wyatt w.r.t. entity.